src/components/PageSections/Menu/_Submenu.scss

```diff
@@ -15,8 +15,7 @@
     & > li .coa-Submenu {
       left: 0;
     }
-    & > li:nth-last-of-type(2) .coa-Submenu,
-    & > li:last-of-type .coa-Submenu {
+    & > li:nth-of-type(n + 6) .coa-Submenu {
```

Comment:
I did this to fix the right to left orientation for menuItem in storybook, but I ended up killing the menuItem storybook component. I left this, because it's simpler to understand.